diff --git a/src/preload/host/sandbox_path.cc b/src/preload/host/sandbox_path.cc
index 95cf0a3..7963b99 100644
--- a/src/preload/host/sandbox_path.cc
+++ b/src/preload/host/sandbox_path.cc
@@ -14,7 +14,7 @@ std::string_view SandboxPath::sandbox_path() const { return sandbox_path_; }
 void SandboxPath::set_sandbox_path(std::string_view path) { sandbox_path_ = path; }
 
 bool SandboxPath::LooksLikeSandboxPath(std::string_view path) {
-  return EndsWith(path, "/chrome-sandbox");
+  return EndsWith(path, "/opera_sandbox");
 }
 
 // static
